packages/eslint-plugin/lib/config/typescript.js

```diff
@@ -69,6 +69,23 @@ module.exports = {
         '@shopify/typescript/prefer-singular-enums': 'error',
         // Prefer buildClientSchema for schema building.
         '@shopify/typescript/prefer-build-client-schema': 'error',
+
+        '@typescript-eslint/naming-convention': [
```

Comment:
Can we note this change in the changelog, please.

Reply:
@ismail-syed I'll do it.